templates/elasticsearch.yml.j2

```diff
@@ -40,6 +40,10 @@ action.auto_create_index: {{ es_action_auto_create_index }}
 xpack.security.enabled: false
 {% endif %}
 
+{% if "monitoring" in es_xpack_features %}
```

Comment:
As discussed in my reply in https://github.com/elastic/ansible-elasticsearch/issues/517#issuecomment-450179055 this goes against best practices to enable by default. Adding it down here would also make it impossible to disable via `es_config` because that variable is at the top of the file.